javascript/javascript2/week1/Shortestword/shortestWord.js

```diff
@@ -0,0 +1,31 @@
+// Method 1 of finding the shortest word in an array
+const danishWords = ['bil', 'plante', 'kaffe', 'bog', 'ø', 'planetarium',[], 0, null, undefined, 'aa', Infinity, true];
+function findShortestWord(word) {
+let shortest = word[0];
+for(let i=0; i < word.length; i++){
+    if (typeof word[i] === 'string' &&
+        (shortest === ''|| word[i].length < shortest.length)) {
+      shortest = word[i];
+    }
+  }
+  
+  return shortest;
+};
+let result = findShortestWord(danishWords);
+console.log(result);
+
+
+// Method 2 of finding the shortest word in a array
+function shortestWord(arr) {
+    let shortest = '';
+    // function to check the array elements length
+    arr.map(function(str){ 
```

Comment:
i believe it's better to use `forEach` and not map. Remember that map always returns an array and you don't need that array

Reply:
implemented the given feedback.